### PR TITLE
update automation readme

### PR DESCRIPTION
**Pull request by sangsoolee**
update automation readme

### CODE REVIEW

File: src/automation/README.md [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
 
-## How to modify
 
 
-## Library
--
+## Overview
 
-## How to make a exec file
+**![automation](https://media.github.ecodesamsung.com/user/18588/files/d0207276-a7a3-4229-97e2-3e71adcd3463)**
```
Comment (junyong-sim) [inline src/automation/README.md:20]: Can we use this link ?

**Review by vedansh-bhardwaj — COMMENTED** [review]
I will check and update.

**Review by vedansh-bhardwaj — COMMENTED** [review]
Looks good, Covered Valuable information about Automation tool.

File: src/automation/README.md (hunk 1 above)
Comment (sangsoolee) [reply src/automation/README.md]: [462eca5](https://github.com/Samsung/ioter/pull/7/commits/462eca53d207ea75c76d7b55e0d6c65380b29990)

**Review by junyong-sim — APPROVED** [review]
LGTM

**Pull request merged**